Pickers/AbstractActionSheetPicker.m

```diff
@@ -540,7 +540,12 @@ - (UIBarButtonItem *)createToolbarLabelWithTitle:(NSString *)aTitle
         textSize = toolBarItemLabel.attributedText.size;
     }
     else {
-        [toolBarItemLabel setTextColor:(NSFoundationVersionNumber > NSFoundationVersionNumber_iOS_6_1) ? [UIColor blackColor] : [UIColor whiteColor]];
+        if (@available(iOS 13.0, *)) { // Support iOS 13 system dark mode
```

Comment:
Please wrap it in as it looks there https://github.com/openid/AppAuth-iOS/pull/412/files#diff-2b222704dbfd09d494d35a28706e0f3fR30-R36

` #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000 `